### ISSUE TITLE
Font for icons in default config

### ISSUE THREAD

**Issue by FivePixels** [Alexays/Waybar#1486]
I'm wondering what the font used in the default configuration file is. When I use the default config, only certain icons show. I just installed ttf-font-awesome-4 from the AUR, but it doesn't seem to make a difference. Some icons work, while others such as hands-free and headset do not.

![image](https://user-images.githubusercontent.com/37427166/158693846-0a26db58-0c68-428c-9687-54453eba8bfd.png)

**Comment by FivePixels** [comment]
I should add, the optional dependency `otf-font-awesome` at the latest version as of today is installed.

**Comment by FivePixels** [comment]
![image](https://user-images.githubusercontent.com/37427166/158696598-da25224f-c589-41a5-b170-a88249fbae0b.png)
Here's an image of what the icons looks like in my bar.

**Comment by Pound-Hash** [comment]
My developer tools shows these icons as part of Apple's set: ui-monospace, Menlo, and SF Mono. Why are these the fonts used in the config? Shouldn't we be using fonts more widely available? Why not just use the recommended fonts: font awesome? Indeed, why not replace the default fonts with ones that are monospaced - the Nerd fonts - and make all the documentation consistently use those same fonts?

**Comment by FivePixels** [comment]
I agree, or even making the dependency a requirement (at least on the AUR), since the code is based on this dependency. Either one sounds like a feasible route.

**Comment by itsKia2** [comment]
On installation, it told me that the font required for the default icons used in the default config  is otf-font-awesome. Perhaps try that out?

**Comment by FivePixels** [comment]
> On installation, it told me that the font required for the default icons used in the default config  is otf-font-awesome. Perhaps try that out?

I already have this installed.

https://github.com/Alexays/Waybar/issues/1486#issuecomment-1069663088
